03-Security-In-CICD/Readme.md

```diff
@@ -98,8 +99,9 @@ When the pipeline executes, SVTs will scan the specified set of resources.
 Along with input parameter, you can check for below options
 <br/>**Enable OMS Logging:** Switch to enable this task to publish SVT evalution results to an OMS workspace. Steps to configure OMS credential are explained in Step-4
 <br/>**Aggregate Control Status:** Switch to aggregate the SVTs control output. When this is turned off it would show all the failed individual controls in the task summary output.
+<br/>**Do not auto-update AzSDK:** Switch to toggle auto update of AzSDK and required AzureRM modules on the build server. Keep this un-checked for Hosted VS2017 and while using SVT task fot the first time.
```

Comment:
And also when ever there you need to update the version of AzSDK on the Non-Hosted Agents. I think you should append this as well.